pkg/api/scylla/v1/types_cluster.go

```diff
@@ -80,6 +80,8 @@ type ScyllaClusterSpec struct {
 
 	// sysctls holds the sysctl properties to be applied during initialization given as a list of key=value pairs.
 	// Example: fs.aio-max-nr=232323
+	// Deprecated: `sysctls` is deprecated. Use NodeConfig to configure sysctls instead.
+	// See https://operator.docs.scylladb.com/stable/api-reference/groups/scylla.scylladb.com/nodeconfigs.html for details.
```

Comment:
I think it could be more future-proof if, instead of linking to a specific page, we mentioned "See NodeConfig resource reference for details.".

Reply:
good point

Reply:
done